### PR TITLE
docs: update landing page

### PR DESCRIPTION
**Pull request by signekb**
## Description

- This PR updates the design landing page. 
- I noticed an unfinished sentence in the hero heading and couldn't let it pass. This resulted in this little update of the landing page. 

## Related Issues

Closes #...

See also Issues #...

## Reviewer Focus

This PR only needs a quick review.

## Checklist

For general documentation:

- [X] Spell-check
- [X] Did the page(s) preview correctly on your machine without breaking
- [X] New category words (keywords) (if any) added to the code snippet file (None)

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
-technical details. It is also used to communicate the goals, general
-structure, requirements, and uses of the software to both technical and
-non-technical people. Along with the design documentation, we also
-document and post the reasons
+This is the first place to look when trying to understand the Seedcase
+framework and how it works from a high-level perspective. 
+
+The design
+documentation is a way of providing an overall map of our software and
+the principles we follow when developing it without getting into highly
```
Comment (K-Beicher) [inline index.qmd:16]: maybe very instead of highly?

File: index.qmd [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+general structure, requirements, and uses of the software to both
+technical and non-technical users and stakeholders.
 :::
 
+
 {{< include /includes/_wip.qmd >}}
 
+## Contents
+
+While the Seedcase framework is software, it also interacts with data,
```
Comment (K-Beicher) [inline index.qmd:27]: There is something that I can't put my finger on with this sentence...

**Review by K-Beicher — DISMISSED** [review]
Just a few things.

File: index.qmd [hunk 3]
```diff
@@ -1,26 +1,48 @@
...
+the code base is open-source, and the user-interface is designed to be
+accessible. The design documentation is divided into four main sections:
+
+-   [Software architecture](software/index.qmd) documentation, which is
+    based on the [arc42](https://arc42.org) template and the [C4
+    Model](https://c4model.com/).
```
Comment (lwjohnst86) [inline index.qmd:33]:
```suggestion
    Model](https://c4model.com/)
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Nice! Only some minor suggestions ☺️

File: index.qmd (hunk 2 above)
Comment (lwjohnst86) [reply index.qmd:27]: I made a suggestion, because I agree.

File: index.qmd (hunk 1 above)
Comment (signekb) [inline index.qmd:16]:
```suggestion
the principles we follow when developing it without getting into
```

**Review by K-Beicher — APPROVED** [review]
Looks good.

**Review by lwjohnst86 — APPROVED** [review]
Nice, much better 🦾

**Pull request merged**